goobi_adapter/goobi_reader/src/test/scala/uk/ac/wellcome/platform/goobi_reader/fixtures/Server.scala

```diff
@@ -0,0 +1,13 @@
+package uk.ac.wellcome.platform.goobi_reader.fixtures
+
+import org.scalatest.Suite
+import uk.ac.wellcome.monitoring.test.fixtures.CloudWatch
+import uk.ac.wellcome.platform.goobi_reader.{Server => AppServer}
+import uk.ac.wellcome.test.fixtures.ServerFixtures
+
+trait Server extends ServerFixtures with CloudWatch { this: Suite =>
+  def newAppServer: () => AppServer = () => new AppServer()
```

Comment:
Not really related to this PR, so not blocking this merge, but I don't really like how `ServerFixtures` works. I don't think we should use polymorphism but composition